### PR TITLE
ci: update helm workflow and repository path

### PR DESCRIPTION
**Pull request by acellard**
Signed-off-by: Alison Cellard <alison.cellard@owkin.com>

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -6,9 +6,8 @@ name: Build
 #  docker: ci/version app tag --insert-dev-info [--ref COMMIT_ID]
 #          (or just replacing + with _ in the app version, basically)
 env:
-    GCP_DOCKER_SERVER: gcr.io
+    REGISTRY: ghcr.io
 # secrets:
-#   GCP_SERVICE_ACCOUNT_KEY_JSON
 #   HELM_REPO_USERNAME
```
Comment (mblottiere) [inline .github/workflows/build.yaml:11]: Can be removed

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -5,9 +5,8 @@ on:
         # FYI this isn't triggered when more than 3 tags are pushed at once
         # https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#push
 env:
-    GCP_DOCKER_SERVER: gcr.io
+    REGISTRY: ghcr.io
 # secrets:
-#   GCP_SERVICE_ACCOUNT_KEY_JSON
 #   HELM_REPO_USERNAME
 #   HELM_REPO_PASSWORD
 #   GH_PAT_USERNAME
```
Comment (mblottiere) [inline .github/workflows/release.yaml:12]: Seems unused as well

File: package.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "name": "substra-frontend",
     "repository": {
         "type": "git",
-        "url": "https://github.com/owkin/connect-frontend.git"
+        "url": "https://github.com/Substra/substra-frontend.git"
     },
     "license": "UNLICENSED",
```
Comment (mblottiere) [inline package.json:7]:
```suggestion
    "license": "Apache-2.0",
```

File: .github/workflows/build.yaml (hunk 1 above)
Comment (acellard) [reply .github/workflows/build.yaml]: All removed

File: package.json (hunk 3 above)
Comment (acellard) [reply package.json]: Changed, thanks!

**Review by alexisdeh — APPROVED** [review]

**Pull request merged**